### PR TITLE
Add LSP2 Coder page

### LINKED ISSUE #21: Add LSP-2 Coder page
similar to ABI Coder page but for all the things related to LSP2 (JSONURL, etc.)

Add "LSP-2 Coder" tab in the navbar.

Then, on this page, there should be one dropdown where the user can select a [`valueContent`](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#valueContent) and a textarea where the user can put the content - either encoded or decoded - and then the user can see the encoded / decoded data.

Use erc725.js for the encoding/decoding:

- create a "fake schema" that you inject in here:

```js
ERC725.encodeData(data, schemas);
```

https://docs.lukso.tech/tools/erc725js/classes/ERC725#encodedata

so the lib encode/decode for you

### PR DESCRIPTION
**Pull request by skimaharvey**
Need to add: 
- valueContent's formatters depending on the valueContent selected
- boolean valueContent's decoder/encoder once it is added to the erc725.js library 
- work on the UI 
- add some common used bytesN valueContents like (bytes4, bytes16, bytes32)

Need to fix: 
- the encoder/decoder for the BitArray because it does not seem to be encoding/decoding correctly

Closes #21

### CODE REVIEW

File: components/ValueTypeDecoder/ValueTypeDecoder.tsx [hunk 1]
```diff
@@ -32,14 +32,14 @@ const ValueTypeDecoder: React.FC<Props> = ({
...
-  const web3 = useWeb3();
+  const { web3 } = useContext(NetworkContext);
 
   useEffect(() => {
     const startDecoding = async () => {
       if (address && web3 !== undefined) {
         const schema: ERC725JSONSchema[] = [erc725JSONSchema];
         const erc725 = new ERC725(schema, address, web3.currentProvider);
-
+        debugger;
```
Comment (CallumGrindle) [inline components/ValueTypeDecoder/ValueTypeDecoder.tsx:42]: debugger left in here

**Review by CallumGrindle — APPROVED** [review]
Super cool. Tested and works well. Only thing I would suggest is to show the input fields before a user has to select a value content, as the inputs are the same for almost all options. Also some conflicts to resolve after network switch was resolved

File: components/Lsp2Coder/Coder.tsx [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+            </div>
+          </div>
+        </div>
+      );
+    } else {
+      return (
+        <textarea
+          placeholder="value"
+          value={decodedValue as string}
+          className="p-1  textarea is-fullwidth"
```
Comment (Hugoo) [inline components/Lsp2Coder/Coder.tsx:70]:
```suggestion
          className="p-1 textarea is-fullwidth"
```

**Comment by skimaharvey** [comment]
Closes #21

**Pull request merged**